db/dev_seeds/polls.rb

```diff
@@ -28,19 +28,49 @@
               ends_at:   1.month.ago,
               results_enabled: true,
               stats_enabled: true)
+
+  Poll.find_each do |poll|
+    name = poll.name
+    I18n.available_locales.map do |locale|
+      neutral_locale = locale.to_s.downcase.underscore.to_sym
```

Comment:
It's no longer necessary to underscore locale names before using them with `Globalize` (and it will actually cause an error), since https://github.com/consul/consul/pull/2896/commits/c7fcdd9b0ec905063a803d93d45a454c826e14db.

The same applies to a few other places.

Reply:
Done https://github.com/consul/consul/pull/2914/commits/888ae8e6d6749d34f358ae21b05fc8f1c8b87d4f